packages/security/carbon.yml

```diff
@@ -0,0 +1,311 @@
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/icon-button-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-iconbuttonbar
+      tags:
+        - input-control
+        - data-display
+  non-entitled-section:
+      name: Non entitled section
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/non-entitled-section.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-nonentitledsection
+      tags:
+        - content-block
+  search-bar:
+      name: Search bar
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/search-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-searchbar
+      tags:
+        - input-control
+  status-indicator:
+      name: Status indictor
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/status-indicator.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-statusindicator
+      tags:
+        - input-control
+  tag-wall:
+      name: Tag wall
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/tag-wall.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-tagwall
+      tags:
+        - input-control
+        - data-display
+  tag-wall-filter:
+      name: Tag wall filter
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/tag-wall-filter.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-tagwallfilter--tagwall-with-filter-list-for-multiselect
+      tags:
+        - input-control
+        - data-display
+  detail:
+      name: Detail
+      status: stable
+      type: template
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/detail.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-layouts-detail
+      tags:
+        - content-block
```

Comment:
Approved. Very cool to see an example of a template-- in the future we'll want to discuss weather the same tags apply as components because every template will be tagged content block.